cms/locale/cy/LC_MESSAGES/django.po

```diff
@@ -451,17 +451,17 @@ msgstr "Methodoleg"
 msgid "Related publications"
 msgstr ""
 
-#: cms/settings/base.py:351
+#: cms/settings/base.py:357
 msgid "English"
 msgstr ""
 
-#: cms/settings/base.py:352
+#: cms/settings/base.py:358
 msgid "Welsh"
 msgstr "Cymraeg"
 
-#: cms/settings/base.py:353
+#: cms/settings/base.py:359
 msgid "Ukrainian"
 msgstr ""
 
-msgid "To be confirmed"
-msgstr "I'w gadarnhau"
+#~ msgid "To be confirmed"
+#~ msgstr "I'w gadarnhau"
```

Comment:
@MaciekBaron we'll need to wrap this in the form validation so it is preserved

Reply:
Thanks @kacperpONS ! Updated :)